descriptors/ModuleDescriptor-template.json

```diff
@@ -116,6 +116,38 @@
         }
       ]
     },
+    {
+      "id": "custom-fields",
+      "version": "1.0",
+      "interfaceType" : "multiple",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/custom-fields",
```

Comment:
Can you rename `/custom-fields` to `/users-custom-fields`?
Each module may have custom fields. The pathPattern of mod-inventory-storage will probably be `/inventory-custom-fields`.
This will result in these URLs:
https://folio-snapshot-stable-okapi.aws.indexdata.com/users-custom-fields
https://folio-snapshot-stable-okapi.aws.indexdata.com/inventory-custom-fields

Reply:
we have defined api for custom fields in [custom fields module](https://github.com/folio-org/mod-custom-fields/blob/master/ramls/custom-fields.raml) and if I change the routing in `mod-users` to `/users-custom-fields` it will simply not work because there is no implementation for such endpoint

Reply:
When Okapi gets a request for 
https://folio-snapshot-stable-okapi.aws.indexdata.com/custom-fields
how should Okapi know whether to route it to mod-users or mod-inventory-storage?
It looks as if https://github.com/folio-org/mod-custom-fields needs to be changed to allow each module having a different pathPattern.

Reply:
by the header `X-Okapi-Module-Id`. Please have a look on my first picture posted in this PR

Reply:
I have already tested it on local vagrant env and posted an image from the real case

Reply:
Yes, you are right. Sorry, I've never seen an implementation of "interfaceType": "multiple" before.